### PR TITLE
added forever: true to request

### PR DESCRIPTION
**Pull request by ericuldall**
Fix for issue #1928

Thanks @martincad for the fix

### CODE REVIEW

File: packages/common/src/util.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ var is = require('is');
 var request = require('request').defaults({
   timeout: 60000,
   gzip: true,
+	forever: true,
```
Comment (stephenplusplus) [inline packages/common/src/util.js]: I think this is a tab, could you replace it with two spaces? Sorry to be *that* guy :(

Comment (ericuldall) [inline packages/common/src/util.js]: Of course.

**Comment by stephenplusplus** [comment]
I believe we need the smallest of additions to our unit tests to make sure `forever: true` is set, right here: https://github.com/GoogleCloudPlatform/google-cloud-node/blob/ee64b07526339dee164de4d4260fc372efc6e78c/packages/common/test/util.js#L95

**Comment by ericuldall** [comment]
I'm waiting on corporate approval for the CLA.

**Comment by stephenplusplus** [comment]
Cool, thanks for going through all of the trouble!

**Comment by ericuldall** [comment]
I signed it! We're under the company name Restoration Media.

**Comment by stephenplusplus** [comment]
Thank you! @jgeewax or @omaray can you confirm?

**Comment by omaray** [comment]
Confirm which part? Using another CLA?

**Comment by stephenplusplus** [comment]
Yes, I don't believe I have visibility to that list. Any confirmation that this PR can be merged.

**Comment by martincad** [comment]
@stephenplusplus What is the steps needed before getting a PR integrated ? Do this need to be approved by a specific amount of reviewers ? 

Are those two failing steps forbidding to approve this PR ? 

Thank you

**Comment by stephenplusplus** [comment]
This PR is actually ready to merge from a technical stance, but I don't have visibility into the signatures on the CLA list. @omaray or @jgeewax can hopefully confirm that this PR is ready to merge. Thanks for your patience!

**Comment by ericuldall** [comment]
I received an email this morning saying that everything was signed.
Is there any way to get some info on the status of our CLA? It doesn't seem that something this trivial should take too long.

**Comment by ericuldall** [comment]
Awesome! I was just about to ask.

**Comment by stephenplusplus** [comment]
Woo, thanks @ericuldall!

**Comment by martincad** [comment]
@stephenplusplus Thank you so much. We are really needing this 👍 

Do you know when a new release is going to be build ? Is it going to be created from master (I mean, is going to include this change ?) 

Thank you so much !

**Pull request merged**